.github/workflows/docker-rust-build.yaml

```
@@ -55,8 +58,9 @@ jobs:
       - uses: imjasonh/setup-crane@v0.1
 
       - name: Build and Push Rust images
-        run: docker/docker-bake-rust-all.sh
+        run: docker/docker-bake-rust-all.sh $DOCKER_BUILD_TARGET
         env:
           PROFILE: ${{ env.PROFILE }}
           FEATURES: ${{ env.FEATURES }}
           BUILD_TEST_IMAGES: ${{ env.BUILD_TEST_IMAGES }}
+          DOCKER_BUILD_TARGET: ${{ env.DOCKER_BUILD_TARGET }}
```

Could we just do this
```suggestion
        run: docker/docker-bake-rust-all.sh ${{ env.DOCKER_BUILD_TARGET }}
        env:
          PROFILE: ${{ env.PROFILE }}
          FEATURES: ${{ env.FEATURES }}
          BUILD_TEST_IMAGES: ${{ env.BUILD_TEST_IMAGES }}
```

---

This needs to be either ${{ inputs.DOCKER_BUILD_TARGET }} or you need to add DOCKER_BUILD_TARGET: ${{ inputs.DOCKER_BIULD_TARGET }} in env?

I dont think this will work as is

---

technically yea, after we import all of them as workflow-level `env`s. lemme try